src/translate/es.ts

```diff
@@ -557,7 +557,8 @@ export default {
     title: "Confirmar pago",
     totalLabel: "Total:",
     totalExceed: "El total excede su saldo de {{balance}}",
-    maxFeeSelected: "Tarifa máxima que se cobrará por esta transacción. Podría ser menor una vez se realice el pago."
+    maxFeeSelected: "Tarifa máxima que se cobrará por esta transacción. Podría ser menor una vez se realice el pago.",
```

Comment:
I think `"Esta es la"` should prefix the translation for `maxFeeSelected`. So it would be `"Esta es la Tarifa máxima que se cobrará por esta transacción. Podría ser menor una vez se realice el pago."`
Cross checked using google translate.

Reply:
@dolcalmi can you confirm?